src/components/QRCodeButton.jsx

```diff
@@ -0,0 +1,59 @@
+// import SwipeableViews from "react-swipeable-views";
+import { makeStyles } from "@material-ui/core/styles";
+import Fab from "@material-ui/core/Fab";
+import { ImQrcode } from "react-icons/im";
+import { Typography } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    alignItems: "center",
+    position: "fixed",
+    top: 140,
+    right: theme.spacing(2),
+    zIndex: 2,
+  },
+  labels: {
+    display: "flex",
+    flexDirection: "column",
+    marginRight: theme.spacing(1),
+  },
+  title: {
+    marginBottom: theme.spacing(-1),
+  },
+  subtitle: {
+    letterSpacing: 0.6,
```

Comment:
Statt unten lieber hier oben spezifizieren? 🤔 
```suggestion
    marginBottom: theme.spacing(-1),
    color: theme.palette.primary.main,
  },
  subtitle: {
    letterSpacing: 0.6,
    color: theme.palette.primary.main,
```

Reply:
Hm mir gefällt eigtl. die Props-Variante. Sind weniger Zeilen/Zeichen 🙂